http-bench-jmh/src/main/scala/org/apache/pekko/BenchTestSource.scala

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
should this file have a Lightbend header?

Reply:
Let me check

Reply:
@He-Pin I will approve this PR if we can work out what license header is needed here.

Reply:
Yes, I copied it from pekko, where it has a lightbend header.